### PR TITLE
157 eg brain region document

### PR DESCRIPTION
**Pull request by egolol**

### CODE REVIEW

File: braph/atlas/BrainRegion.m [hunk 1]
```diff
@@ -1,4 +1,31 @@
 classdef BrainRegion < handle & matlab.mixin.Copyable
+    % BrainRegion < handle & matlab.mixin.Copyable: A brain region.
+    % BrainRegion contains the information of a brain region.
+    %
+    % BrainRegion properties (GetAccess=protected, SetAccess=protected):
+    %   label       -    unique identifier for the brain region.
```
Comment (giovannivolpe) [inline braph/atlas/BrainRegion.m]: align all "-"
also after - use a single space

File: braph/atlas/BrainRegion.m [hunk 2]
```diff
@@ -8,7 +35,15 @@
     end
     methods
         function br = BrainRegion(label, name, x, y, z)
+            % BrainRegion(varargin) creates a barin region with
```
Comment (giovannivolpe) [inline braph/atlas/BrainRegion.m]: barin -> brain

File: braph/atlas/BrainRegion.m [hunk 3]
```diff
@@ -8,7 +35,15 @@
...
+            % default properties for each missing argument.
```
Comment (giovannivolpe) [inline braph/atlas/BrainRegion.m]: remove  "for each missing argument."

File: braph/atlas/BrainRegion.m (hunk 2 above)
Comment (giovannivolpe) [inline braph/atlas/BrainRegion.m]: remove varargin

File: braph/atlas/BrainRegion.m [hunk 4]
```diff
@@ -29,37 +64,93 @@
...
+            
             label = br.label;
         end
         function name = getName(br)
+            % GETNAME returns the name of the brain region.
+            %
+            % STRING = GETNAME(BR) returns the extendend name of the brain
+            % region.
+            %
+            % See also getLabel(), getX(), getPosition().
```
Comment (giovannivolpe) [inline braph/atlas/BrainRegion.m]: add also getY() and getZ()

Also below, where relevant

File: braph/atlas/BrainRegion.m [hunk 5]
```diff
@@ -29,37 +64,93 @@
                 label = 'BR';
             end
             
-            br.label = label;
+            % initialization of brain region propeties.
```
Comment (annacanal) [inline braph/atlas/BrainRegion.m]: properties

File: braph/atlas/BrainRegion.m [hunk 6]
```diff
@@ -1,4 +1,31 @@
...
+    % Graph methods:
+    %   BrainRegion  - Constructor.
+    %   tostring     - returns a string representing the brain region.
+    %   disp         - displays the brain region.
+    %   getLabel     - returns the label.
+    %   getName      - returns the name.
+    %   getX         - returns the x coordinate.
+    %   getY         - returns the y coordinate.
+    %   getZ         - returns the z coordinate.
+    %   getPosition  - retuns an array with the x, y, z coordinates.
```
Comment (annacanal) [inline braph/atlas/BrainRegion.m]: returns. missing r

**Pull request merged**